language/stdlib/modules/LibraConfig.move

```diff
@@ -164,6 +167,27 @@ module LibraConfig {
         include SetEnsures<Config>;
     }
 
+    fun disable_reconfiguration(lr_account: &signer) {
+        assert(
+            Signer::address_of(lr_account) == CoreAddresses::LIBRA_ROOT_ADDRESS(),
+            Errors::requires_address(ELIBRA_CONFIG)
+        );
+        Roles::assert_libra_root(lr_account);
+        move_to(lr_account, DisableReconfiguration {} )
+    }
+
+    fun enable_reconfiguration(lr_account: &signer) acquires DisableReconfiguration {
+        assert(Signer::address_of(lr_account) == CoreAddresses::LIBRA_ROOT_ADDRESS(), Errors::requires_address(ELIBRA_CONFIG));
+
+        if(exists<DisableReconfiguration>(Signer::address_of(lr_account))) {
+            DisableReconfiguration {} = move_from<DisableReconfiguration>(Signer::address_of(lr_account));
+        }
+    }
+
+    fun should_reconfigure(): bool {
```

Comment:
This name is a bit confusing--maybe `reconfiguration_enabled`?

Reply:
Updated in fixup!